enterprise/backend/src/metabase_enterprise/audit_db.clj

```diff
@@ -181,7 +181,8 @@
   :default    true
   :visibility :internal
   :setter     :none
-  :audit      :never)
+  :audit      :never
+  :doc        false)
```

Comment:
Excluding internal-only env var from the docs (unrelated change but tiny so rolling it into this PR)